sdk/quantum/quantum-jobs/rollup.base.config.js

```diff
@@ -0,0 +1,171 @@
+import nodeResolve from "@rollup/plugin-node-resolve";
+import multiEntry from "@rollup/plugin-multi-entry";
+import cjs from "@rollup/plugin-commonjs";
+import replace from "@rollup/plugin-replace";
+import { terser } from "rollup-plugin-terser";
+import sourcemaps from "rollup-plugin-sourcemaps";
+import shim from "rollup-plugin-shim";
+import json from "@rollup/plugin-json";
+import * as path from "path";
+import inject from "@rollup/plugin-inject";
+
+const pkg = require("./package.json");
+const depNames = Object.keys(pkg.dependencies);
+const input = "dist-esm/src/index.js";
+const production = process.env.NODE_ENV === "production";
+
+const version = pkg.version;
+const banner = [
+  "/*!",
+  " * Copyright (c) Microsoft and contributors. All rights reserved.",
+  " * Licensed under the MIT License. See License.txt in the project root for",
+  " * license information.",
+  " * ",
+  ` * Azure Quantum Jobs SDK for JavaScript - ${version}`,
+  " */"
+].join("\n");
+
+const ignoreKnownWarnings = (warning) => {
+  if (warning.code === "THIS_IS_UNDEFINED") {
+    // This error happens frequently due to TypeScript emitting `this` at the
+    // top-level of a module. In this case its fine if it gets rewritten to
+    // undefined, so ignore this error.
+    return;
+  }
+
+  if (
+    warning.code === "CIRCULAR_DEPENDENCY" &&
+    warning.importer.indexOf(path.normalize("node_modules/chai/lib") === 0)
+  ) {
+    // Chai contains circular references, but they are not fatal and can be ignored.
+    return;
+  }
+
+  console.error(`(!) ${warning.message}`);
+};
+
+export function nodeConfig(test = false) {
+  const externalNodeBuiltins = ["events", "crypto", "path"];
+  const baseConfig = {
+    input: input,
+    external: depNames.concat(externalNodeBuiltins),
+    output: {
+      file: "dist/index.js",
+      format: "cjs",
+      sourcemap: true,
+      banner: banner,
+      name: "azurequantumjobs"
+    },
+    preserveSymlinks: false,
+    plugins: [
+      sourcemaps(),
+      replace({
+        delimiters: ["", ""]
+      }),
+      nodeResolve({ preferBuiltins: true }),
+      cjs()
+    ]
+  };
+
+  if (test) {
+    // Entry points - test files under the `test` folder(common for both browser and node), node specific test files
+    baseConfig.input = ["dist-esm/test/*.spec.js", "dist-esm/test/**/*.spec.js"];
```

Comment:
Does this `"dist-esm/test/**/*.spec.js"` not cover all the test files? Meaning.. does it that not include this `"dist-esm/test/*.spec.js"`?
(I don't know.. I'm asking!)

Reply:
It does not cover.